playbooks/roles/logger/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+logger_address: 192.168.9.90
```

Comment:
We should rename the `logger` role to be more revealing; server vs client; sender vs receiver.

Maybe `rsyslog-client` and `rsyslog-server` or `logstash-client` and `logstash-host` or some such..